rules/DeadCode/Rector/Stmt/RemoveUnreachableStatementRector.php

```diff
@@ -105,6 +105,8 @@ private function processCleanUpUnreachabelStmts(array $stmts): array
 
             $previousStmt = $stmts[$key - 1];
 
+            // unset...
```

Comment:
it uses array_splice, so the comment `// unset...` is invalid? as php documentation : 

```
array_splice — Remove a portion of the array and replace it with something else
```

Reply:
In case of `array_splice($items, $key)` with no new array, it only removes the array item.

Reply:
ok 👍